lib/ansible/modules/network/avi/avi_api_version.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
+"""
+# Created on July 24, 2017
+#
+# @author: Vilian Atmadzhov (vilian.atmadzhov@paddypowerbetfair.com) GitHub ID: vivobg
+#
+# module_check: not supported
+#
+# Copyright: (c) 2017 Gaurav Rastogi, <grastogi@avinetworks.com>
+#                     Vilian Atmadzhov, <vilian.atmadzhov@paddypowerbetfair.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+"""
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: avi_api_version
+author: Vilian Atmadzhov (vilian.atmadzhov@paddypowerbetfair.com)
+
+short_description: Avi API Version Module
+description:
+    - This module can be used to obtain the version of the Avi REST API. U(https://avinetworks.com/)
+version_added: 2.5
+requirements: [ avisdk ]
+options:
+    tenant:
```

Comment:
This isn't defined in the argspec

Reply:
@gundalow wasn't sure if options is required. I have it taken out in next commit.

Reply:
If the module doesn’t doesn’t have any options (for example, it’s a _facts module), you can use `options: {}`.

From http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html